VERSION

```diff
@@ -1 +1 @@
-0.3.0
+0.4.0
```

Comment:
0.3.0-criteo1 ?

You can also get the version from the tag: https://github.com/criteo-forks/cassandra/commit/c780ab223d538a04cda99b5ac8acbef517f7ee77#diff-354f30a63fb0907d4ad57269548329e3R13